.github/workflows/gh-pages.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     if: github.event_name != 'pull_request'
     runs-on: ubuntu-latest
     env:
-      BUILD: '*'
+      BUILD: '**'
```

Comment:
I'm not sure if this is the way to go. This will import all descendants that have a config, including descendants that are already included in the config of their parent. For example the tutorials are gathered in a parent tutorial config, here this pattern would cause duplicate imports for all tutorials right?

Reply:
I mean for now it doesn't matter, since there is nothing yet in the build directory, but still.

Reply:
I think I want the same structure for versioned and unversioned docs. So I will just go for `*/*/documentation`

Reply:
```suggestion
      BUILD: '*/*/documentation'
```